### ISSUE TITLE
Validation loss in tensorboard log remains 0

### ISSUE THREAD

**Issue by wvalcke** [ultralytics/ultralytics#242]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and found no similar bug report.

### YOLOv8 Component

Training

### Bug

Using the simple command yolo to to test training:
If you then check tensorboard graphs, the loss of the validation remains a flat line at 0.
The yolov5 scripts did this correct.
Special option needed?

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by AyushExel** [comment]
@wvalcke Hi! can you confirm if you're on the latest version on ultralytics package by running `pip install --upgrade ultralytics`? Then let us know if you still see this problem. Thanks!

**Comment by wvalcke** [comment]
Hi Ahush,

I did the upgrade, and retrained. The problem remains, also you can see this in the results.csv file.
val/box_loss remains at zero. val/cls_loss is inf, and val/dfl_loss is 0.
The lr values (pg0, pg1, pg2) are filled in the csv, but are missing in the tensorboard logs. In yolov5 those values were also logged.

**Comment by AyushExel** [comment]
This issue might've been solved in the latest release. Please upgrade and let us know if you face any problems
